components/Main.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Image from "next/image";
+import MainLogo from "../public/MainLogo.png";
+import styles from "../styles/Main.module.css";
+
+const Main = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.containerMain}>
+        <div className={styles.title}>
+          <h1>
+            Cada <span className="text-[#5651e5]"> Junior </span>es
```

Comment:
```suggestion
            Cada <span className={styles.text}>Junior</span> es
```

same for the other styles

Reply:
Move these colors to custom colors

https://tailwindcss.com/docs/customizing-colors#using-custom-colors

Reply:
@joselyepez don't forget to change this